app.js

```diff
@@ -3,18 +3,15 @@ document.addEventListener("DOMContentLoaded", () => {
     timestamp: Date.now(),
```

Comment:
The brief says `creating an array of currency rate objects` so you need to create more objects like `currencyData` and store them in an array. This will have an impact on your search function.

Reply:
@vdaguenet I have now implemented the search function as instructed. I also made  some improvement to the functions in general.